lib/rexml/element.rb

```diff
@@ -15,9 +15,256 @@ module REXML
   # context node and convert it back when we write it.
   @@namespaces = {}
 
-  # Represents a tagged XML element.  Elements are characterized by
-  # having children, attributes, and names, and can themselves be
-  # children.
+  # An \REXML::Element object represents an XML element.
+  #
+  # An element:
+  #
+  # - Has a name (string).
+  # - May have a parent (another element).
+  # - Has zero or more children (other elements, text, CDATA, and comments).
+  # - Has zero or more siblings (other elements, text, CDATA, and comments).
```

Comment:
Could you also add processing instructions to children and siblings?

Reply:
Done.